.github/workflows/linux.yml

```diff
@@ -30,28 +31,35 @@ jobs:
     - name: Install Ubuntu dependencies
       run: |
         # NOTE: not the same OpenBLAS version as in upstream CI (I'm being lazy here)
-        sudo apt-get install libopenblas-dev libatlas-base-dev liblapack-dev gfortran libgmp-dev libmpfr-dev libsuitesparse-dev ccache libmpc-dev
+        sudo apt-get update
+        sudo apt-get -yq -o Debug::pkgProblemResolver=true -o Debug::Acquire::http=true install libopenblas-dev libatlas-base-dev liblapack-dev gfortran libgmp-dev libmpfr-dev libsuitesparse-dev ccache libmpc-dev
```

Comment:
What is going on with `Debug::pkgProblemResolver=true -o Debug::Acquire::http=true` here? Is that needed for `act`?

Reply:
Unfortunately, yes. This was suggested by one of the engineers working on `act` to fix our Ubuntu dependencies installation issue.